rest-twitter/service/application.py

```diff
@@ -0,0 +1,104 @@
+from flask import Flask, request, jsonify
+import pymongo
+from bson.json_util import dumps
+from datetime import datetime 
+import json 
+import computeSent
+
+# Connect to Mongo Store
+client = pymongo.MongoClient("mongodb://jpvm:jpvmtdb@18.191.169.163/tweets_db")
+
+
+# Use tweets database
+db = client.tweets_db 
+
+# Use sentiment collection
+sentiments = db.sentiment_collection 
+
+
+# Elastic Beanstalk app setup
+
+# EB looks for an 'application' callable by default
+application = Flask(__name__)
+
+# function to get current timestamp used for get endpoint query parameters
+def get_timestamp():
+    return datetime.now().strftime(("%Y-%m-%d %H:%M:%S"))
+
+
+# Decorator for index page of endpoint
+@application.route('/')
+def home():
+    return '''<h1>Go to /trendsentiment</h1>'''
+
+
+
+# Main function for handling GET and POST
+@application.route('/trendsentiment', methods=['GET', 'POST'])
```

Comment:
Could be `@application.route('/trendsentiment', methods=['GET'])`

Create separate handlers for different methods

Reply:
created two separate handlers